libs/shared/components/navbar/src/lib/navbar.component.ts

```diff
@@ -85,4 +86,29 @@ export class NavbarComponent implements OnInit
       this.router.navigate(['/admin-reports']);
     }
   }
+
+  explore(){
+    if (this.kids < 1){
+      this.openToast('You need to have children added to your profile in order to hire an Au Pair');
```

Comment:
Does the length of the sentence work okay with our toast?

Reply:
This is copied and pasted from our other toast from dashboard

Reply:
Okay cool should be good then